frontend/app/actions/package.scala

```diff
@@ -14,6 +15,8 @@ package object actions {
 
   type GoogleAuthRequest[A] = AuthenticatedRequest[A, googleauth.UserIdentity]
 
+  val countryGroupKey = "country-group"
```

Comment:
So this is referenced in views and such but doesn't seem to be read here, am I missing something?
ideally it'd be in the same place that reads it